echo/echo_integration_test.go

```diff
@@ -87,14 +87,16 @@ func startServer(t *testing.T) string {
 	go func() {
 		err := cmd.Execute(ctx, system)
 		if err != nil {
-			t.Fatal(err)
+			panic(err)
```

Comment:
Do these changes fix the echo test failures?

Reply:
Doesn't fix it, but allows the error to be properly propagated to the test runner. Because members of `t` can't be called from gorotounes (but panic can)